app/views/group_offers/_form.html.slim

```diff
@@ -72,6 +62,31 @@
     .col-xs-12
       = f.input :schedule_details
 
+  h3= t_attr(:volunteers)
+  .row
+    .col-xs-12.col-md-4
+      = f.input :necessary_volunteers, collection: 1..10
+    - if @group_offer.volunteers.any?
+    .col-xs-12.col-md-4
+      =t_attr(:volunteers)
+      ul
+        - @group_offer.volunteers.each do |volunteer|
+          li= link_to volunteer, volunteer_path(volunteer)
+    .col-xs-12.col-md-4
+      button.btn.btn-primary(data-toggle='collapse' data-target='#volunteerBlock' type='button'  aria-expanded='false' aria-controls='volunteerBlock')
+        - if action_name ==  'new'
```

Comment:
It could be also the case that when created volunteers weren't added and they are as part of an edit action. So what checked could be `if action_name == 'new' || @group_offer.empty?`